tests/webapp/api/test_push_api.py

```diff
@@ -407,23 +408,30 @@ def test_push_search(client, test_repository):
     Commit.objects.create(
         push=push1, revision="1234abcd", author="kaz <foo@bar.com>", comments="Initial commit"
     )
+
     Commit.objects.create(
         push=push2, revision="2234abcd", author="foo <foo2@bar.com>", comments="Bug 12345567 - fix"
     )
+
     Commit.objects.create(
         push=push3,
         revision="3234abcd",
         author="quxzan <qux@bar>.com",
         comments="Bug 12345567 - Feature added",
     )
-
+    Commit.objects.update(
+        search_vector=SearchVector("revision", "author", "comments", config="english")
+    )
     # Test search by comments
     resp = client.get(
         reverse("push-list", kwargs={"project": test_repository.name}) + "?search=bug"
     )
     assert resp.status_code == 200
 
     results = resp.json()["results"]
+
+    print(list(Commit.objects.values("revision", "author", "comments", "search_vector")))
```

Comment:
`Substr` for `comments` should still be necessary.

Reply:
I've added `Substr` to the migration

Reply:
Hi @Archaeopteryx  when you have the time, can you review the PR? I've updated the recommended changes and rebased branch